src/rt-app.c

```diff
@@ -849,6 +849,108 @@ static void set_thread_affinity(thread_data_t *data, cpuset_data_t *cpu_data)
 	}
 }
 
+#if HAVE_LIBNUMA
+static int create_numaset_str(numaset_data_t *numa_data)
+{
```

Comment:
witha event based policy you don't need this anymore

Reply:
Will remove.